src/utils/utils.ts

```diff
@@ -138,7 +138,7 @@ export function getCheckBoxesFromTriggerDefinition(definitions: any): checkBox[]
         // restrict possible checkbox values for simplicity
         // - custom checkbox has only two possible values, but not supported by 'change'
         // - group === requester also not easily determined
-        if (action.values && action.group === 'ticket' && action.subject !== 'follower' && !isCustomField) {
+        if (action.values && action.group === 'ticket' && action.subject !== 'follower' && !isCustomField && action.subject !== 'brand_id') {
```

Comment:
Would it help to make these strings into constants?

Reply:
This will get revamped in v2 of the plugin.  Ticket created so this doesn't get lost
https://mattermost.atlassian.net/browse/MM-35365